n3fit/src/evolven3fit_new/evolve.py

```diff
@@ -1,27 +1,27 @@
-import pathlib
 import logging
+import pathlib
 import sys
 
 import numpy as np
-from reportengine.compat import yaml
-
-from ekobox import genpdf, gen_info
-from ekomark import apply
+import eko
 from eko import basis_rotation as br
```

Comment:
```suggestion
from eko import basis_rotation
```

Not this PR, but basis_rotation is a bit more descriptive

Reply:
It is, we are just using the abbreviation in EKO since it is used over and over in some contexts (and its members are already telling enough).
But here it is also fine to use the full name :)

Reply:
I indeed assumed the abbreviation it was an eko thing, but in NNPDF/nnpdf it doesn't make much sense to me.